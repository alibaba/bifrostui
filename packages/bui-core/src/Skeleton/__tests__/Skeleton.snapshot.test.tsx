import { snapshotTest } from 'testing';

describe('Ranting demo snapshot', () => {
  snapshotTest('Skeleton');
});
