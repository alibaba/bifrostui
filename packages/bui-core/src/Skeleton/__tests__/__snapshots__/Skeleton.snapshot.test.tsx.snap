// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Ranting demo snapshot Skeleton demo snapshot 0 1`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <span
    className="bui-skeleton bui-skeleton-text bui-skeleton-animation-wave"
    style={
      Object {
        "height": undefined,
        "width": 210,
      }
    }
  />
  <span
    className="bui-skeleton bui-skeleton-text bui-skeleton-animation-wave"
    style={
      Object {
        "height": undefined,
        "width": 240,
      }
    }
  />
</div>
`;

exports[`Ranting demo snapshot Skeleton demo snapshot 0 2`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <span
    className="bui-skeleton bui-skeleton-text bui-skeleton-animation-wave"
    style={
      Object {
        "height": undefined,
        "width": 210,
      }
    }
  />
  <span
    className="bui-skeleton bui-skeleton-circular bui-skeleton-animation-wave"
    style={
      Object {
        "height": 48,
        "width": 48,
      }
    }
  />
  <span
    className="bui-skeleton bui-skeleton-rectangular bui-skeleton-animation-wave"
    style={
      Object {
        "height": 40,
        "width": 210,
      }
    }
  />
  <span
    className="bui-skeleton bui-skeleton-rounded bui-skeleton-animation-wave"
    style={
      Object {
        "height": 40,
        "width": 210,
      }
    }
  />
</div>
`;

exports[`Ranting demo snapshot Skeleton demo snapshot 0 3`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <span
    className="bui-skeleton bui-skeleton-text bui-skeleton-animation-wave"
    style={
      Object {
        "height": undefined,
        "width": 210,
      }
    }
  />
  <span
    className="bui-skeleton bui-skeleton-text bui-skeleton-animation-pulse"
    style={
      Object {
        "height": undefined,
        "width": 210,
      }
    }
  />
  <span
    className="bui-skeleton bui-skeleton-text"
    style={
      Object {
        "height": undefined,
        "width": 210,
      }
    }
  />
</div>
`;

exports[`Ranting demo snapshot Skeleton demo snapshot 0 4`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "0",
    }
  }
>
  <div
    style={
      Object {
        "alignItems": "center",
        "display": "flex",
        "marginBottom": "12px",
      }
    }
  >
    <span
      className="bui-skeleton bui-skeleton-circular bui-skeleton-animation-wave"
      style={
        Object {
          "height": "48px",
          "margin": "0 6px 0 0",
          "width": "48px",
        }
      }
    />
    <div>
      <span
        className="bui-skeleton bui-skeleton-text bui-skeleton-animation-wave"
        style={
          Object {
            "height": undefined,
            "width": "210px",
          }
        }
      />
      <span
        className="bui-skeleton bui-skeleton-text bui-skeleton-animation-wave"
        style={
          Object {
            "height": undefined,
            "width": "210px",
          }
        }
      />
    </div>
  </div>
</div>
`;

exports[`Ranting demo snapshot Skeleton demo snapshot 0 5`] = `
<div
  className="bui-stack"
  style={
    Object {
      "--align-items": "center",
      "--flex-direction": "column",
      "--flex-wrap": "nowrap",
      "--justify-content": "center",
      "--spacing": "10px",
    }
  }
>
  <span
    className="bui-skeleton bui-skeleton-circular bui-skeleton-animation-wave bui-skeleton-with-children"
    style={
      Object {
        "height": undefined,
        "width": undefined,
      }
    }
  >
    <div
      style={
        Object {
          "height": "80px",
          "width": "80px",
        }
      }
    />
  </span>
  <span
    className="bui-skeleton bui-skeleton-circular bui-skeleton-animation-wave bui-skeleton-with-children"
    style={
      Object {
        "height": undefined,
        "width": undefined,
      }
    }
  >
    <div
      style={
        Object {
          "height": "100px",
          "width": "100px",
        }
      }
    />
  </span>
</div>
`;
